src/scss/components/events-highlights.scss

```diff
@@ -0,0 +1,83 @@
+.event {
+  $root: &;
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 80px;
+
+  &.feature {
+    padding-bottom: 100px;
+    #{$root}__title {
+      font-size: $mobile-title;
+      order: -1;
+    }
+  }
+
+  > * {
+    margin: 0 auto;
+    max-width: 425px;
+    padding: 0 19px;
+    width: 100%;
+  }
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-card-titles;
+    padding-bottom: 20px;
+  }
+
+  &__image {
+    padding-bottom: 20px;
+  }
+
+  &__text {
+    color: $gray;
+    font-family: $hkgrotesk-regular;
+    font-size: $mobile-paragraph;
+    padding-bottom: 40px;
+  }
+
+  &__more-info {
+    .more-info {
+      display: flex;
+      flex-direction: column;
```

Comment:
These two properties are not being applied because you are not using a class `.more-info` inside the `.event__more-info` class

Reply:
So, you can remove them